primitives/avail/src/lib.rs

```diff
@@ -64,6 +64,8 @@ pub enum InvalidTransactionCustomId {
 	ForbiddenAppId,
 	/// Max padded length was exceeded.
 	MaxPaddedLenExceeded,
+	/// Max recursion was reached for a call with AppId != 0.
+	MaxRecursionExceeded,
```

Comment:
Need a corresponding PR for `avail` repo, mainly about the error code section in the readme, if not anything else. 
https://github.com/availproject/avail#transaction-custom-ids

Reply:
Yeah, here's the corresponding PR with the readme updated: 
https://github.com/availproject/avail/pull/177